sapi/phpdbg/phpdbg_frame.c

```diff
@@ -269,42 +267,45 @@ void phpdbg_dump_backtrace(size_t num) /* {{{ */
 		return;
 	} phpdbg_end_try_access();
 
-	Z_LVAL(startline) = zend_get_executed_lineno();
-	startfilename = zend_get_executed_filename();
-	Z_STR(startfile) = zend_string_init(startfilename, strlen(startfilename), 0);
-
-	zend_hash_internal_pointer_reset_ex(Z_ARRVAL(zbacktrace), &position);
+	line = zend_get_executed_lineno();
+	file = zend_get_executed_filename_ex();
```

Comment:
Unlike `zend_get_executed_filename`, this can return NULL.

Reply:
I'm not sure why that is a problem? `file` is always guarded by a `NULL` check.

Reply:
Right, nvm